prepare-solvcon-dev.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+#
+# Usage:
+#   source <this script>
+#   or
+#   source <this script> <your-project-folder>
+#
+set -x
+SOLVCON_PROJECT=${1:-${HOME}/work-my-projects/solvcon}
+SCSRC="${SOLVCON_PROJECT}/solvcon"
+SCSRC_WORKING="${SOLVCON_PROJECT}/solvcon-working"
+MINICONDA_DIR="${SCSRC_WORKING}/miniconda"
+
+export PATH="${SCSRC}:${MINICONDA_DIR}/bin:${PATH}"
+
+mkdir -p ${SCSRC_WORKING}
+
+# fetch SOLVCON source
+git clone git@github.com:solvcon/solvcon.git ${SCSRC}
+
+# prepare miniconda
+# please note contrib/conda.sh will use python 3.6 if you want to install
+# paraview with this conda 3.6 python, this issue may happen to you github
+# https://github.com/conda-forge/paraview-feedstock/issues/27 by using 3.8
+# python conda this issue is resolved by the recompiled paraview package and
+# the corresponding libraries
+#
+# install paraview via conda-forge:
+#     conda install -c conda-forge paraview
+wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ${SCSRC_WORKING}/miniconda.sh
```

Comment:
It seems only work on linux, maybe need to check what OS used by user?

Reply:
Interesting, would you please paste error messages if any?

Besides, what is the canonical way for a macOS user to check "if this is a mac"? I could definitely google that but I want to know the first hand user experience : )

Reply:
Not sure about the canonicality, but it works for me: https://github.com/yungyuc/hack/blob/master/build.d/python/build-python.sh#L49

I think we should set up CI / github action for checking in too.  If not with this PR, we can do it in a separate one.

Reply:
@tai271828 And I think it's self-explaining that the installer works only with Linux, for it has "`Linux`" in the file name.  It seems to me that no error messages are needed for fixing that issue.

Reply:
Yes, the installer works only with **Linux**, due to the miniconda's url specified **Linux** 
maybe can add some code to check what OS used by user:
```
case $(uname) in
  Linux*)
    platform=Linux
    ;;
  Darwin*)
    platform=MacOSX
    ;;
esac
wget https://repo.continuum.io/miniconda/Miniconda3-latest-$(platform)-x86_64.sh -O ${SCSRC_WORKING}/miniconda.sh
```
So that wget can download the right version of miniconda.

Reply:
In terms of downloader, CURL might be a more popular choice than wget.  Some systems have CURL but not wget.